eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -59,8 +59,9 @@ import NoThunks.Class (NoThunks (..))
 import Validation (failureUnless)
 
 data GovEnv era = GovEnv
-  { teTxId :: !(TxId (EraCrypto era))
-  , teEpoch :: !EpochNo
+  { geTxId :: !(TxId (EraCrypto era))
+  , geEpoch :: !EpochNo
+  , geGovActionExpiry :: !EpochNo
```

Comment:
I think it is better to just pass all of PParams, since we'll need some of the other ones in the GOV rule in the future. Namely `minCommitteSize` and `govActionDeposit`
```suggestion
  , gePParams :: !PParams era
```